app/src/main/res/values/strings.xml

```diff
@@ -137,5 +137,12 @@
     <string name="title_confirm_delete_timeline">Are you sure to delete this post ?</string>
 
     <string name="action_edit">Edit</string>
+    <string name="action_change_password">Change Password</string>
+    <string name="title_current_password">Current Password</string>
+    <string name="title_new_password">New Password</string>
+    <string name="title_confirm_password">Confirm Password</string>
+    <string name="title_OK">OK</string>
+    <string name="title_CANCEL">CANCEL</string>
```

Comment:
WARNING: [ButtonCase] The standard Android way to capitalize CANCEL is "Cancel" (tip: use `@android:string/cancel` instead)
 The standard capitalization for OK/Cancel dialogs is "OK" and "Cancel". To ensure that your dialogs use the standard strings, you can use the resource strings @android:string/ok and @android:string/cancel.